docs/resources/scenarios/analyze-web-downloads.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Office Scripts sample scenario: Internet Traffic Analysis'
+description: 'A sample that takes raw internet traffic data in an Excel workbook and determines the origin location, before organizing that information into a table.'
+ms.date: 02/18/2020
+localization_priority: Normal
+---
+
+# Office Scripts sample scenario: Analyze web downloads
+
+In this scenario, you are tasked with analyzing download reports from your company's website. The goal of this analysis is to determine if the web traffic is coming from the United States or elsewhere in the world.
+
+Your colleagues upload the raw data to your workbook. Each week's set of data has its own worksheet. There is also the **Summary** worksheet with a table and chart that shows week-over-week trends.
```

Comment:
Consider rewording:

Your colleagues upload the raw data to your workbook. Each week's set of data has its own worksheet.

->Your colleagues upload the raw data to your workbook, where each worksheet contains one week of data.